charts/argocd-apps/values-test.yaml

```diff
@@ -745,3 +745,37 @@ applications:
           secretKeyRef:
             name: email-alert-auth
             key: token
+- name: smart-answers
+  helmValues:
+    appImage:
+      repository: 172025368201.dkr.ecr.eu-west-1.amazonaws.com/smart-answers
+      tag: latest  # To be edited by automation (not yet implemented).
+    replicaCount: 1
+    extraEnv:
+      - name: EXPOSE_GOVSPEAK
+        value: 1
+      - name: COMPANIES_HOUSE_API_KEY
```

Comment:
Just double checked and realised Smart Answers no longer uses this (we never implemented the functionality that used the API). I've raised a [PR to remove it from puppet](https://github.com/alphagov/govuk-puppet/pull/11564) - but it'd also be safe to remove from here too.

Reply:
Oh haha, thanks! Fewer secrets is always good :D